src/mlpack/methods/cf/decomposition_policies/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Define the files we need to compile
+# Anything not in this list will not be compiled into mlpack.
+set(SOURCES
+  batch_svd_method.hpp
+  nmf_method.hpp
+  randomized_svd_method.hpp
+  regularized_svd_method.hpp
+  SVD_complete_method.hpp
```

Comment:
Do you mind to use ``svd_complete_method`` and ``svd_incomplete_method.hpp``, to be consistent with the rest of the codebase?